openid-provider-commands-1_0.md

```diff
@@ -218,15 +225,24 @@ The following Claims are used within the Command Token:
 
 - **tenant**  
   REQUIRED.  
-  A JSON string. The Tenant identifier. MAY have the value `personal`, `organization` or a stable OP unique value for multi-tenant OPs. The `personal` value is reserved for when Accounts are managed by individuals. The `organization` value is reserved for Accounts are managed by an organization.
+  A JSON string. The OP's identifier for the OP's tenant. See {{OpenID.Enterprise}}.
+  
+  MUST have the value `personal`, `organization` or a stable OP unique value for multi-tenant OPs. The `personal` value is reserved for when Accounts are managed by individuals. The `organization` value is reserved for Accounts that are managed by an organization.
 
   The combination of `iss` and `tenant` uniquely identifies a Tenant. 
 
 - **callback_token**  
   OPTIONAL in an Asynchronous Command and the `metadata` command.  
   An OP generated unique and opaque token for the RP to use when calling the OP's **callback_endpoint** Asynchronous Command responses and making a metadata refresh request.
+
+- **aud_sub**  
+  OPTIONAL for Account Commands.  
+  The RP's internal identifier for the Account, as provided by the RP during account resolution. When present, the RP SHOULD use this value to identify the Account rather than the combination of `iss` and `sub`. This claim facilitates more efficient account lookup and management at the RP.
```

Comment:
```suggestion
  A JSON string.  The RP's internal identifier for the Account, as provided by the RP during account resolution. When present, the RP SHOULD use this value to identify the Account rather than the combination of `iss` and `sub`. This claim facilitates more efficient account lookup and management at the RP.
```

Reply:
change made